prueba/src/anagram/AnagramIdentifier.java

```diff
@@ -0,0 +1,24 @@
+package anagram;
+
+public class AnagramIdentifier {
+
+	public boolean anagramDetect(String word1, String word2) {
+
+		if (word1.length() != word2.length()) {
+			return false;
+		}
+		int j;
+		int countEqualLetters = 0;
+
+		for (int i = 0; i < word1.length(); i++) {
+			j = word2.length() - i - 1;
+			if (word1.charAt(i) == word2.charAt(j)) {
+				countEqualLetters++;
+			}
+		}
+		if ((countEqualLetters == word1.length())) {
```

Comment:
And also, I have a better idea! :D
You can do this and skip this if:
```java
return countEqualLetters == word1.length();
```

As that operation returns a boolean value, if it meets the condition, it will return true, otherwise false.
You are doing exactly the same functionality without using the if statement :) and it looks cleaner :P